src/status_im2/contexts/wallet/create_account/view.cljs

```diff
@@ -1,16 +1,79 @@
 (ns status-im2.contexts.wallet.create-account.view
-  (:require [react-native.core :as rn]
-            [quo2.core :as quo]
-            [re-frame.core :as rf]))
+  (:require
+    [quo2.theme :as quo.theme]
+    [react-native.core :as rn]
+    [quo2.core :as quo]
+    [react-native.safe-area :as safe-area]
+    [reagent.core :as reagent]
+    [status-im2.contexts.wallet.common.temp :as temp]
+    [utils.i18n :as i18n]
+    [status-im2.contexts.wallet.create-account.style :as style]
+    [utils.re-frame :as rf]))
 
-(defn view
+(defn- view-internal
   []
-  [rn/view
-   {:style {:flex            1
-            :align-items     :center
-            :justify-content :center}}
+  (let [top                  (safe-area/get-top)
+        bottom               (safe-area/get-bottom)
+        color                (reagent/atom :blue)
```

Comment:
i think it's far beyond this pr at this point but we should really put this default of `:blue` in a contstant somewhere 😅

Reply:
also consider `account-color` in place of `color` as it is a bit more descriptive of the use case

Reply:
Yes we should have a default `account-color` constant

Reply:
oh good point, it's slightly different than the rest of the app for this use of color. but every other use of the default `customization-color` is hardcoded. I'll make an issue to fix it!